app/data/__tests__/known_accounts.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable testing-library/await-async-query */
```

Comment:
[note]

`findByName` function is declared in the file.

There are `findBy...` functions in React Testing Library, which are supposed to return Promise.
see: https://testing-library.com/docs/queries/about/

The eslint rules enforces Promise should be returned from `findBy...`.

Since RTL is not used (will not be used) in the file, I ignored it.
But we can change the function name if you like.

Reply:
ignoring in this case is fine.